src/odemis/acq/feature.py

```
@@ -281,6 +281,43 @@ def save_milling_task_data(self,
         logging.info(f"Stage position for milling: {self.get_posture_position(MILLING)}")
         logging.info(f"Feature {self.name.value} is ready to mill.")
 
+def feature_decoder(feature_raw):
+    """
+    Json decoder for the CryoFeature class and its attributes
+    """
+    # Either the object is the feature list or the feature objects inside it
+    if not 'name' in feature_raw or not 'status' in feature_raw:
```

Ruff (0.15.10)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)